packages/now-static-build/src/index.ts

```diff
@@ -493,8 +493,34 @@ export async function build({
       const extraOutputs = await readBuildOutputDirectory({ workPath });
 
       // No need to verify the dist dir if there are other output files.
```

Comment:
This comment is in the wrong place. It can probably be removed

Reply:
👍 Moved it to the `validateDistDir` function. I'd rather keep it in case someone wonders why we ignore the validation when `functions` is present.